plugin/modelgen/models.gotpl

```diff
@@ -29,7 +29,7 @@
 			{{- with .Description }}
 				{{.|prefixLines "// "}}
 			{{- end}}
-			{{ $field.Name|go }} {{$field.Type | ref}} `{{$field.Tag}}`
```

Comment:
So is this because of a bug in the `go` filter?

Reply:
Nope, the `go` filter was just being erroneously applied in situations where the field name and its capitalization were being explicitly overridden in the gqlgen config. This was a problem because that filter changes the capitalization of field names that are already valid in Go in a slightly opinionated way, which was unwanted behavior in this case.

Reply:
Got it. Cool.

Reply:
@ianling Is there a way (in config or otherwise) to disable this?

Reply:
@YogeshLele disable what exactly? If you want to specify a particular capitalization for a field in your schema, this PR made it so you can do so without gqlgen overriding it.